### ISSUE TITLE
Calendar - re-visit previous/next month dates styling

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet-theme-hpe#416]
The text color of the previous/next month dates doesn't meet contrast ratio requirements. We should explore how to style these dates so that they are differentiated from the current month dates but still meet contrast ratios.

**Comment by jcfilben** [comment]
Work handled in https://github.com/grommet/grommet/issues/7423. Follow on work tracked here: https://github.com/grommet/grommet/issues/7467
